### ISSUE TITLE
API Error

### ISSUE THREAD

**Issue by oxhagolli** [NotoriousRebel/TimeWeather-App#1]
I downloaded the project and followed the instructions on the Readme.md - I get the following error in the console:
```
Failed to load resource: the server responded with a status of 401 (Unauthorized)
https://api.openweathermap.org/data/2.5/weather?zip=98052&cnt=1&APPID=
```

I assume APPID missing is why this is failing. Is there a workaround?

**Comment by NotoriousRebel** [comment]
This has been resolved in #2

**Comment by NotoriousRebel** [comment]
As this has been resolved, closing.
